test/Directory.Packages.props

```diff
@@ -0,0 +1,138 @@
+<Project>
+  <Import Project="..\Directory.Packages.props" />
+
+  <!-- TODO: I'm having second thought on whether it would be better to consolidate all packages in one ItemGroup and sort them alphabetically. -->
+  <!-- TODO: Take special care for the complicated logic in TestApplication.MongoDB.csproj -->
```

Comment:
Better to leave the `TestApplication.MongoDB.csproj` untouched. It will make it difficult to read/understand.

Reply:
Done.